requirements.txt

```diff
@@ -6,6 +6,7 @@ promise==2.3
 pydantic==1.7.3
 Rx==3.1.1
 six==1.15.0
-starlette==0.14.1
+starlette==0.13.6
```

Comment:
What's the reason for the version downgrade?

Reply:
fast API doesnt support starlette 0.14.1